packages/concerto-analysis/src/compare-utils.ts

```diff
@@ -43,3 +43,13 @@ export function getPropertyType(property: Property) {
         throw new Error(`unknown property type "${property}"`);
     }
 }
+
+export function getValidatorType(validator: Validator) {
+    if (validator instanceof NumberValidator) {
```

Comment:
Probably not applicable here - but we've been bitten by using instanceof for exported types, as instanceof also checks the module for the type.